libnavui-androidauto/build.gradle

```diff
@@ -40,8 +40,10 @@ dependencies {
 
     // This defines the minimum version of Navigation which is included in this SDK. To upgrade the
     // Navigation versions, you can specify a newer version in your downstream build.gradle.
-    releaseApi("com.mapbox.navigation:android:2.9.0-alpha.4")
-    debugApi project(":libnavigation-android")
+    // TODO reset to a specific version when the release is ready (and make license)
+    api project(":libnavigation-android")
```

Comment:
Why not do this only for debug? Also it won't be necessary to regenerate licence file this way.

Reply:
it failed unit tests https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/21334/workflows/0d5f5f2f-aa5d-4357-bfc8-9ffdd39f9966/jobs/109591